src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -453,21 +453,24 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 </PropertyGroup>
 
 <Target Name="_BuildAdditionalResourcesCache"
-	Inputs="@(ReferencePath);@(ReferenceDependencyPaths);$(MSBuildProjectFullPath);$(NugetPackagesConfig);$(_AndroidBuildPropertiesCache)"
-	Outputs="$(_AndroidResourcePathsCache)"
-	DependsOnTargets="$(_BeforeBuildAdditionalResourcesCache)"
-	>
- <GetAdditionalResourcesFromAssemblies
-   AndroidSdkDirectory="$(_AndroidSdkDirectory)"
-   AndroidNdkDirectory="$(_AndroidNdkDirectory)"
-   Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
-   CacheFile="$(_AndroidResourcePathsCache)"
-   YieldDuringToolExecution="$(YieldDuringToolExecution)"
-   DesignTimeBuild="$(DesignTimeBuild)"
-   ContinueOnError="$(DesignTimeBuild)"
-   Condition=" '$(DesignTimeBuild)' == '' Or '$(DesignTimeBuild)' == 'false' "
+    Inputs="@(ReferencePath);@(ReferenceDependencyPaths);$(MSBuildProjectFullPath);$(NugetPackagesConfig);$(_AndroidBuildPropertiesCache)"
+    Outputs="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp"
+    DependsOnTargets="$(_BeforeBuildAdditionalResourcesCache)"
+    Condition=" '$(DesignTimeBuild)' != 'True' ">
+  <GetAdditionalResourcesFromAssemblies
+      AndroidSdkDirectory="$(_AndroidSdkDirectory)"
+      AndroidNdkDirectory="$(_AndroidNdkDirectory)"
+      Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
+      CacheFile="$(_AndroidResourcePathsCache)"
+      YieldDuringToolExecution="$(YieldDuringToolExecution)"
+      DesignTimeBuild="$(DesignTimeBuild)"
+      ContinueOnError="$(DesignTimeBuild)"
   />
-  <Message Text="Skipping GetAdditionalResourcesFromAssemblies in DesignTime build" Condition=" '$(DesignTimeBuild)' == 'true' " />
+  <Touch Files="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp" AlwaysCreate="True" />
+  <ItemGroup>
+    <FileWrites Include="$(_AndroidResourcePathsCache)" Condition=" Exists ('$(_AndroidResourcePathsCache)') " />
+    <FileWrites Include="$(_AndroidStampDirectory)_BuildAdditionalResourcesCache.stamp" />
+  </ItemGroup>
 </Target>
```

Comment:
I formatted this target because the whitespace was pretty crazy:
- Some spots were indented with 3 spaces?
- There were randomly a couple of tabs?